00-HTML-CSS-basics/Multimedia.html

```diff
@@ -21,8 +23,8 @@ <h2>Video</h2>
 <br><br>
 <h2>Sound + image</h2>
 
-<img src="https://image.freepik.com/free-icon/play-button_318-42541.jpg" width="50" onClick="document.getElementById('audio_play').play(); return false;" />
-<img src="https://image.freepik.com/iconen-gratis/video-pauzeknop_318-33989.jpg"width="50" onClick="document.getElementById('audio_play').pause(); return false;" />
+<img alt="img-Play" src="https://image.freepik.com/free-icon/play-button_318-42541.jpg" width="50" onClick="document.getElementById('audio_play').play(); return false;" />
```

Comment:
Is good that you actually specified an `alt` value for the images, but I'd rather put a more user friendly message there since it will be a backfall in case the image don't load.

Anyway, do not adress this issue since is not required, the concept is fine.